src/components/TestingTemplates/index.tsx

```diff
@@ -0,0 +1,213 @@
+                <IconArrowForward className="margin-left-1" />
+              </span>
+            </UswdsLink>
+          </li>
+          <li className="margin-bottom-1">
+            <UswdsLink href="#test-plan">
+              <span className="display-flex flex-align-center">
+                {t('testingTemplates.testPlanSection.heading')}
+                <IconArrowForward className="margin-left-1" />
+              </span>
+            </UswdsLink>
+          </li>
+          <li className="">
+            <UswdsLink href="#remediation-plan">
+              <span className="display-flex flex-align-center">
+                {t('testingTemplates.remediationPlanSection.heading')}
+                <IconArrowForward className="margin-left-1" />
+              </span>
+            </UswdsLink>
+          </li>
+        </ul>
+      </div>
+      <div className="templates-for-508-testing__section">
+        <h2 id="vpat" className="margin-bottom-2">
+          {t('testingTemplates.vpatSection.heading')}
+        </h2>
+        <p className="margin-bottom-4">
+          {t('testingTemplates.vpatSection.description')}
+        </p>
+        <h3 className="margin-bottom-2">
+          {t('testingTemplates.vpatSection.subSection.heading')}
+        </h3>
+        <p className="margin-bottom-1">
+          {t('testingTemplates.vpatSection.subSection.item1.text')}
+        </p>
+        <ul className="usa-list usa-list--unstyled desktop:grid-col-8">
+          {vpatConformanceLevels.map(level => (
+            <li
+              key={level.name}
+              className="maxw-none margin-bottom-1  padding-left-2"
+            >
+              <span className="text-bold display-inline">{level.name}</span>{' '}
+              {level.description}
+            </li>
+          ))}
+        </ul>
+        <p className="margin-bottom-4">
+          {t('testingTemplates.vpatSection.subSection.item2.text')}
+        </p>
+        <SummaryBox
+          heading={t(
+            'testingTemplates.vpatSection.subSection.downloadVPAT.heading'
+          )}
+          className="desktop:grid-col-6 margin-bottom-6"
+        >
+          <p>
+            <UswdsReactLink
+              to="https://www.itic.org/policy/accessibility/vpat"
+              target="_blank"
+              rel="noopener noreferrer"
+              variant="external"
+            >
+              {t(
+                'testingTemplates.vpatSection.subSection.downloadVPAT.line1.linkText'
+              )}
+            </UswdsReactLink>
+            {` `}
+            {t(
+              'testingTemplates.vpatSection.subSection.downloadVPAT.line1.otherText'
+            )}
+          </p>
+          <p className="display-flex flex-row flex-align-center">
+            <IconError className="margin-right-1" />
+            {` `}
+            {t(
+              'testingTemplates.vpatSection.subSection.downloadVPAT.line2.text'
+            )}
+          </p>
+          <p>
```

Comment:
Wrapping this link in a `<p>` tag is adding extra margin below this link, so you could remove it altogether or just reduce the margin.